rts/motoko-rts-tests/src/mark_stack.rs

```diff
@@ -32,12 +32,12 @@ fn test_<M: Memory>(mem: &mut M, n_objs: u32) -> TestCaseResult {
         alloc_mark_stack(mem);
 
         for obj in &objs {
-            push_mark_stack(mem, *obj as usize);
+            push_mark_stack(mem, *obj as usize, obj.wrapping_sub(1));
```

Comment:
is that an unskew?

Reply:
Not really, I just wanted a value derived from `obj` for the `tag` argument of `push_mark_stack`. Since the actual value does not matter (push/pop does not care about validness of tags) I more or less randomly used `obj - 1`.

Reply:
Oh, never mind, I didn't realize this was just dummy data.

Reply:
(should have looked at the context)